Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -129,6 +129,13 @@
   icon: /Textures/Objects/Tools/emag.rsi/icon.png
   price: 8
 
+- type: uplinkListing
+  id: UplinkHypopen
+  category: Utility
+  itemId: Hypopen
+  description: A chemical hypospray disguised as a pen, capable of instatly injecting up to 15u of reagents. Starts empty.
+  price: 2
```

Comment:
2 seems low for what is essentially a free hypospray

Reply:
I playtested it, with a 15u capacity, the only thing which would bring a person in crit is amanita toxin, literally every other harmful reagent is not a kill with 15u

Reply:
Now thinking back on it, it's too reliable as a finisher with bicardine and a few punches/shanks with a knife, I'll bump it to 4